GetLimitedProfile/__tests__/handler.test.ts

```diff
@@ -88,10 +93,14 @@ describe("GetLimitedProfileHandler", () => {
           const mockProfileModel = ({
             findLastVersionByModelId: jest.fn(() => taskEither.of(none))
           } as unknown) as ProfileModel;
+          const mockServicesPreferencesModel = ({
+            getQueryIterator: jest.fn(() => fromLeft({}))
+          } as unknown) as ServicesPreferencesModel;
```

Comment:
Why not extracting single costs definitions and eventually override its implementation with `mockImplementationOnce` ?

Reply:
done.